js/main.js

```diff
@@ -0,0 +1,35 @@
+function getrandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
```

Comment:
Меняешь входные параметры, лучше заведи доп переменные
```suggestion
  const to = Math.floor(max);
```